apps/src/lab2/views/components/Instructions/ResourcePanel/VersionHistory/VersionHistoryPanel.tsx

```diff
@@ -355,14 +360,23 @@ const VersionHistoryPanel: React.FunctionComponent<
         });
       }
       if (viewingInitialVersion) {
+        // Should onRestore be passed as an arg?
         dispatch(previewStartSources({startSources}));
+        if (onRestore) onRestore(startSources);
       } else if (isLatest) {
-        dispatch(resetToCurrentVersion());
+        if (onRestore) dispatch(resetToCurrentVersion({onRestore}));
```

Comment:
should `onRestore` be required here to call `resetToCurrentVersion`? As-is will this cause broken behavior in Web Lab 2/Python lab?